package/kedro_viz/api/rest/responses.py

```diff
@@ -452,7 +457,7 @@ def save_api_pipeline_response_to_fs(pipelines_path: str, remote_fs: Any):
             raise exc
 
 
-def save_api_responses_to_fs(path: str, remote_fs: Any):
+def save_api_responses_to_fs(path: str, remote_fs: Any, are_datasets_previewable: bool):
```

Comment:
This is personal preference, but I would change `are_datasets_previewable` to `preview_datasets`.

To me `are_datasets_previewable` sounds like it's checking whether it's possible to preview these datasets or not, where when reading the code it should mean whether preview has been enabled or not. 

In my proposed change `preview_datasets=True` means preview is enabled and `preview_datasets=False` means it's disabled.

Reply:
okay, I too had a hard time renaming this :) .... so I want to make it differentiable to the `is_preview_disabled` which corresponds to each instance of DataNodeMetadata class, where-as the plural form `are_datasets_previewable` is a class variable. I personally prefer having is/are before a boolean variable/method to be clear of the return/data type.